src/main/java/org/verdictdb/jdbc41/VerdictStatement.java

```diff
@@ -42,10 +42,67 @@ public VerdictStatement(VerdictConnection jdbcConn, VerdictContext context) {
     this.executionContext = context.createNewExecutionContext();
   }
 
+  /**
+   * Check whether given sql contains 'bypass' keyword at the beginning
+   *
+   * @param sql original sql
+   * @return without 'bypass' keyword if the original sql begins with it. null otherwise.
+   */
+  private String checkBypass(String sql) {
```

Comment:
Can we place this logic inside VerdictContext? The Java applications that directly use VerdictContext would not support BYPASS in this way.

Reply:
Sounds good to me.